test/fixtures/linter/autofix/bigquery/004_templating/before_4226.sql

```diff
@@ -0,0 +1,8 @@
+{%- set vals = "has_used_small_subject_line\n ,has_used_personal_subject\n ,has_used_personal_to\n" -%}
```

Comment:
What's the purpose of this file?

Reply:
Ooops - that was a test for a different PR. I'll remove it.